packages/core/src/state/NetworkManager.ts

```diff
@@ -29,6 +34,7 @@ export default class NetworkManager implements Manager {
   declare readonly dataExpiryLength: number;
   declare readonly errorExpiryLength: number;
   protected declare middleware: Middleware;
+  private lastClear = new Date(-1);
```

Comment:
seems like maybe doing `new Date(NaN)` would be a clearer choice?

Reply:
new Date(NaN) < new Date(0) === false

Whatever is on left hand side I want to always be less than right hand side.

Reply:
in that case, just `-Infinity`? ie, not a date object at all

Reply:
Oh nice, didn't realize that had such a property 👍